### DISCUSSION TITLE
GDScript Implementation

### DISCUSSION THREAD

**Discussion by TsFreddie** [sqids/sqids-spec#22] [General]
Happened to need Sqids in the [Godot Engine](https://github.com/godotengine/godot). 

Here's my implementation: https://github.com/TsFreddie/sqids-gdscript

The contribution guide `https://sqids.org/faq#contribute` did not specify what to do for language that does not have a todo repo. So I figured I could submit here.

**Comment by 4kimov** [comment]
Hi @TsFreddie, that's great - thanks for pinging about that! I'm sure others will find it useful too.
Would you like to push it to https://github.com/sqids/sqids-gdscript while I get the website ready? I sent you an invite for that repo.

**Reply by TsFreddie** [reply]
I've pushed to the repo. Thanks for the invite.

I also want to submit to the [Godot Asset Library](https://godotengine.org/asset-library/). The repo is already setup to conform with the [Submission Guideline](https://docs.godotengine.org/en/stable/community/asset_library/submitting_to_assetlib.html). Let me know if there is any concern about that and maybe how you want to approach the submitting process.

**Reply by 4kimov** [reply]
Regarding submitting: go ahead and submit it since you're the expert on this. If there's any kind of sqids.org DNS entries needed, no problem - just let me know and I'll add whatever record is necessary.

The [website page is up](https://sqids.org/gdscript), let me know if I messed anything up in examples or anywhere else.

Thanks for doing all this work 💪
